assets/src/edit-story/panels/shared.js

```diff
@@ -9,6 +9,13 @@ import PropTypes from 'prop-types';
  */
 import { __ } from '@wordpress/i18n';
 
+export const RowPanel = styled.form`
```

Comment:
In the near future, the panel has a lot more functionality than just a wrapping form, so don't replace it with a different panel. The size panel should still use `Panel` as the main wrapper. In stead create a new row element that goes inside a panel as this row will have to go together with other options as can be seen in the design.

Reply:
New Row and SpaceSpan components created on shared with some basic styling props.